doc/docs/master/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -66,20 +97,43 @@ Where:
 
 | Parameter     | Description | 
 | ------------- |-------------| 
-| **protocol**   | The name of the database protocol. As of today, you have the ability to connect to  `postgres`or `mysql`. Note that `mysql` allows you to connect to mariadb. |
+| **protocol**   | The name of the database protocol. <br> As of today, we support: <br>- `postgres` and `postgresql` : connect to Postgresql or compatible (for example Redshift).<br>- `mysql` : connect to MySQL or compatible (for example MariaDB). <br>- `snowflake` : connect to Snowflake. |
 | **username**  | The user used to access the database.|
 | **host**      | The hostname of your database instance.|
 | **port**      | The port number your instance is listening on.|
 | **database**  | The name of the database to connect to. | 
 
+
+
+!!! Attention "Snowflake users, you will need a variant of the url above."
+     Pachyderm supports two connection URL patterns to query Snowflake: 
+
+      - `snowflake://username@<account_identifier>/<db_name>/<schema_name>?warehouse=<warehouse_name>`
+      - `snowflake://username@hostname:port/<db_name>/<schema_name>?account=<account_identifier>&warehouse=<warehouse_name>`
+
+     where:
+
+      - *[`account_identifier`](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html)* takes the following form for most URLs:
+
+        - short:  `organization_name`-`account_name`.
+        - long: If you are used to connecting to Snowflake via an URL such as `https://organization_name-account_name.snowflakecomputing.com`, you can use the full domain name `organization_name-account_name.snowflakecomputing.com`.
```

Comment:
I think we should also note that we support the account-locator style as well. https://docs.snowflake.com/en/user-guide/admin-account-identifier.html#label-account-locator

So `account_locator.region_id.cloud`

Reply:
Essentially, there are two ways to specify the `account_identifier`:

1. Account Name: `<organization_name>-<account_name>`
2. Account Locator: `<account_locator>.<region>.<cloud>`

The `.snowflakecomputing.com` is optional for both